sortinghat/cmd/unify.py

```diff
@@ -117,13 +124,16 @@ def unify(self, matching=None, sources=None,
         :param fast_matching: use the fast mode
         :param no_strict_matching: disable strict matching (i.e, well-formed email addresses)
         :param interactive: interactive mode for merging identities
+        :param recovery: if enabled, the unify will read the matching identities stored in
+           recovery file (`~/.sortinghat/unify.log`) and process them
```

Comment:
This is not the file where data is stored. I think it is better to say `stored in recovery file (RECOVERY_FILE_PATH)`